app/vmselect/main.go

```diff
@@ -342,6 +342,18 @@ func selectHandler(qt *querytracer.Tracer, startTime time.Time, w http.ResponseW
 	}
 
 	if strings.HasPrefix(p.Suffix, "prometheus/vmalert") {
+		if strings.HasSuffix(p.Suffix, "/") {
```

Comment:
Comments from [here](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/2845/files#r916692051) are still relevant to this change.